packages/external-integration/e2e/ExternalPaymentMethod.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    PaymentStepAsGuestPreset,
+    test,
+} from '@bigcommerce/checkout/payment-integration-test-framework';
+
+import { internalOrder, order, orderPayment } from './MockResponse';
+
+test.describe('External payment method', () => {
+    test('Sezzle is working', async ({ assertions, checkout, page }) => {
+        // Testing environment setup
+        const responseProps = { status: 200, contentType: 'application/json' };
+
+        await checkout.use(new PaymentStepAsGuestPreset());
+        await checkout.start('Sezzle');
+        await checkout.route(
+            'https://sandbox.checkout.sezzle.com?id=**',
+            `${__dirname}/support/sezzle.com.ejs`,
+        );
```

Comment:
This test is similar to Google Pay and Apple Pay, mockups needed.
When everything goes as plan, the checkout page will turn to `https://sandbox.checkout.sezzle.com?id=**`, then come back to the order confirmation page.